registration/app/registration/services/NotificationRegistrarProvider.scala

```diff
@@ -16,12 +16,16 @@ case class UnsupportedPlatform(platform: String) extends RequestError {
   override def reason: String = s"Platform '$platform' is not supported"
 }
 
-final class NotificationRegistrarProvider(windowsNotificationRegistrar: WindowsNotificationRegistrar, gcmNotificationsRegistrar: GCMNotificationRegistrar)
+final class NotificationRegistrarProvider(
+  windowsNotificationRegistrar: WindowsNotificationRegistrar,
+  gcmNotificationRegistrar: GCMNotificationRegistrar,
+  apnsNotificationRegistrar: APNSNotificationRegistrar)
   (implicit executionContext: ExecutionContext) extends RegistrarProvider {
 
   override def registrarFor(registration: Registration): NotificationsError \/ NotificationRegistrar = registration match {
```

Comment:
maybe that would look nicer if the pattern match was happening on `registration.platform`?